src/core_plugins/region_map/public/region_map_vis_params.html

```diff
@@ -1,45 +1,49 @@
-<div class="form-group">
-
+<div class="kuiSideBarSection">
   <div class="kuiSideBarSectionTitle">
-    <div class="kuiSideBarSectionTitle__text">
-      Layer Settings
-    </div>
+    <div class="kuiSideBarSectionTitle__text">Options</div>
```

Comment:
maybe we should just strip this. Looks better without

![image](https://user-images.githubusercontent.com/1833023/31201360-67e3497a-a92c-11e7-9cbe-fe4e9af5db6e.png)